values.sample.yaml

```diff
@@ -28,18 +10,47 @@ tonicdb: {
   sslMode: Require
 }
 
-numberOfWorkers: 1
-
 environmentName: <company-name>
 
-enableLogCollection: false
 
+# tonicStatisticsSeed if set will cause generations to be consisent with each other for anything where consistency is set to true.
+# otherwise each generation will be internally consistent, but not consistent with other generations.
```

Comment:
We don't really have the option of no stats seed rn though

Reply:
I'm not sure I agree. isn't the absence of this variable the "no stats seed" option?

Reply:
Maybe? My understanding of helm charts is low.
We're using it like this
`
        - name: TONIC_STATISTICS_SEED
          value: {{quote .Values.tonicStatisticsSeed }}
`
If they delete the line to set it, does null gets passed in and then we interpret it as no stats seed?
Or do we need some sort of if statement around it so that we don't pass anything?

Reply:
I added an if statement to see if it is defined or not. Good catch!